frontends/mit-open/src/page-components/SearchDisplay/SearchDisplay.tsx

```diff
@@ -293,17 +441,54 @@ const SearchDisplay: React.FC<SearchDisplayProps> = ({
     },
     { keepPreviousData: true },
   )
+
+  const [mobileDrawerOpen, setMobileDrawerOpen] = React.useState(false)
+
+  const toggleMobileDrawer = (newOpen: boolean) => () => {
+    setMobileDrawerOpen(newOpen)
+  }
+
+  const filterContents = (
+    <>
+      <FacetStyles>
+        <ProfessionalToggle
+          professionalSetting={requestParams.professional}
+          setParamValue={setParamValue}
+        ></ProfessionalToggle>
+        <AvailableFacets
+          facetManifest={facetManifest}
+          activeFacets={requestParams}
+          onFacetChange={toggleParamValue}
+          facetOptions={data?.metadata.aggregations ?? {}}
+        />
+      </FacetStyles>
+    </>
+  )
+
+  const sortDropdown = (
+    <StyledDropdown
+      initialValue={requestParams.sortby || ""}
+      isMultiple={false}
+      onChange={(e) => setParamValue("sortby", e.target.value)}
+      options={SORT_OPTIONS}
+      renderValue={(value) => {
+        const opt = SORT_OPTIONS.find((option) => option.key === value)
+        return `Sort by: ${opt?.label}`
+      }}
+    />
+  )
+
   return (
     <Container>
-      <GridContainer>
+      <StyledGridContainer>
         <ResourceTypeTabs.Context
           resourceType={requestParams.resource_type?.[0]}
         >
-          <GridColumn variant="sidebar-2-wide-main">
+          <DesktopFiltersColumn variant="sidebar-2">
             <FacetsTitleContainer>
               <FilterTitle>
-                <Typography variant="h5">Filters</Typography>
-                <TuneIcon fontSize="inherit" />
+                <Typography variant="h5">Filter</Typography>
+                <TuneIcon fontSize="medium" />
```

Comment:
This replaced by the Remix Icon "equalizer-line" icon, https://remixicon.com/icon/equalizer-line